index.html

```diff
@@ -0,0 +1,424 @@
+            </svg>
+            <h1>E-Cards</h1>
+        </div>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#">Transactions</a>
+                </li>
+                <li>
+                    <a href="#">Cards</a>
+                </li>
+            </ul>
+        </nav>
+        <div>
+            <img src="img/profile_logo.png" alt="">
+            <div>
+                <span>Itai Bracha</span>
+                <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+            </div>
+        </div>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#">Settings</a>
+                </li>
+                <li>
+                    <a href="#">Logout</a>
+                </li>
+            </ul>
+        </nav>
+    </aside>
+    
+    <section>
+            <h2>Cards</h2>
+        <section>
+            <div>
+                <div>
+                    <div>
+                        <h3>Current Balance</h3>
+                    </div>
+                    <div>
+                        <p>VISA</p>
+                    </div>
+                </div>
+                <div>
+                    <p>$4,570,80</p>
+                </div>
+                <div>
+                    <p>5294 2436 4780 2468</p>
+                </div>
+                <div>
+                    <div> 
+                        <div>    
+                            <p>NAME</p>
+                        </div>
+                        <div>
+                            <p>Itai Bracha</p>
+                        </div>
+                    </div>
+                    <div>
+                        <div>         
+                            <p>Valid Thru</p>
+                        </div>
+                        <div>     
+                            <p>12/24</p>
+                        </div>    
+                    </div>
+                    <div>
+                        <div>     
+                            <p>cvv</p>
+                        </div>
+                        <div>     
+                            <p>344</p>
+                        </div>    
+                    </div>    
+                </div>    
+                </div>
+            </div>
+            <div>
+                <button>Add new Card

```

Comment:
```suggestion
                            <div>
                <button>
                    <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
                        <path
                            d="M16 0C7.17706 0 0 7.17706 0 16C0 24.8229 7.17706 32 16 32C24.8229 32 32 24.8217 32 16C32 7.17831 24.8229 0 16 0ZM16 29.5213C8.54531 29.5213 2.47869 23.4559 2.47869 16C2.47869 8.54406 8.54531 2.47869 16 2.47869C23.4547 2.47869 29.5213 8.54406 29.5213 16C29.5213 23.4559 23.4559 29.5213 16 29.5213Z"
                            fill="#6B6F7B" />
                        <path
                            d="M22.1967 14.6491H17.2394V9.69174C17.2394 9.00761 16.6854 8.45236 16 8.45236C15.3146 8.45236 14.7606 9.00761 14.7606 9.69174V14.6491H9.80325C9.11787 14.6491 8.56387 15.2044 8.56387 15.8885C8.56387 16.5726 9.11787 17.1279 9.80325 17.1279H14.7606V22.0852C14.7606 22.7694 15.3146 23.3246 16 23.3246C16.6854 23.3246 17.2394 22.7694 17.2394 22.0852V17.1279H22.1967C22.8821 17.1279 23.4361 16.5726 23.4361 15.8885C23.4361 15.2044 22.8821 14.6491 22.1967 14.6491Z"
                            fill="#6B6F7B" />
                    </svg>
                    <p>Add new Card</p>
                </button>
            </div>

```
You need to put text before the icon since it is placed above it